src/fmu/sumo/explorer/objects/polygons.py

```diff
@@ -16,13 +16,16 @@ def __init__(self, sumo: SumoClient, metadata: Dict) -> None:
         """
         super().__init__(sumo, metadata)
 
-    def to_dataframe(self) -> pd.DataFrame:
+
+    def to_pandas(self) -> pd.DataFrame:
         """Get polygons object as a DataFrame
 
         Returns:
             DataFrame: A DataFrame object
         """
+        warn('.to_dataframe() is deprecated, renamed to .to_pandas() ', DeprecationWarning, stacklevel=2)
```

Comment:
Should probably move this warning to a `.to_dataframe()`?
Something like:
```
def to_dataframe(self) -> pd.DataFrame:
   warn(`The warning`)
   return self.to_pandas()
```

Reply:
and similar for `.to_pandas()` and `.to_arrow()` in `/objects/table.py`

Reply:
@equinor-ruaj here 44b986e0de7e6b2a2c55af76f823c5a07e4375ef